src/OpenTelemetry.Instrumentation.GrpcNetClient/CHANGELOG.md

```diff
@@ -9,6 +9,11 @@
   [issue](https://github.com/open-telemetry/opentelemetry-dotnet/issues/5092)
   for details and workaround.
   ([#5077](https://github.com/open-telemetry/opentelemetry-dotnet/pull/5077))
+* Removed support for `http` and `http/dup` values for`OTEL_SEMCONV_STABILITY_OPT_IN`
```

Comment:
`OTEL_SEMCONV_STABILITY_OPT_IN` - this entire env variable is now ignored right? Maybe make it explicitly clear that this env var will be ignored now.